code/utils/variance-per-voxel.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np 
+import matplotlib.pyplot as plt 
+import nibabel as nib
+
+def load_image(fname):
+	"""
+	Loads the .nii file 
+
+	Parameters
+	----------
+	fname : String
+	The path to the .nii file that we wish to load and get the data of
+
+	Returns
+	-------
+	data : a 4-D array
+	"""
+	img = nib.load(fname)
+	data = img.get_data()
+	return data
```

Comment:
Should we remove the first 4 volumes since we never want to use that data anyway?  That also affects our module on scene time and location